### PR TITLE
CSEN-94 Integration tests were implemented for new requests

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/it/scala/com/bwsw/cloudstack/entities/util/requests/RequestExecutionHandler.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package com.bwsw.cloudstack.entities.util.requests
+
+import br.com.autonomiccs.apacheCloudStack.exceptions.ApacheCloudStackClientRequestRuntimeException
+import com.bwsw.cloudstack.entities.TestEntities
+import com.bwsw.cloudstack.entities.requests.Request
+
+import scala.util.{Failure, Success, Try}
+
+object RequestExecutionHandler extends TestEntities {
+  def doesEntityNotExist(request: Request): Boolean = {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/util/requests/RequestExecutionHandler.scala]: entityNotExist

File: src/it/scala/com/bwsw/cloudstack/entities/requests/domain/DomainFindRequestIntegrationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        addParameter(Constants.ParameterKeys.NAME, name)
+      }
+    }
+    domainFindRequest.withName(domainName)
+    val response = mapper.deserialize[DomainFindResponse](executor.executeRequest(domainFindRequest.getRequest))
+
+    assert(response.entityList.entities.isEmpty)
+  }
+
+  it should "retrieve json string if request contains default parameters and parameter with incorrect key" in {
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/entities/requests/domain/DomainFindRequestIntegrationTestSuite.scala]: ignore a parameter with incorrect key

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/entities/util/requests/RequestExecutionHandler.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/util/requests/RequestExecutionHandler.scala]: ok

**Review by placccebo — APPROVED** [review]

File: src/it/scala/com/bwsw/cloudstack/entities/requests/domain/DomainFindRequestIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/entities/requests/domain/DomainFindRequestIntegrationTestSuite.scala]: ok

**Pull request merged**